sdk/cardboard.cc

```diff
@@ -344,6 +344,17 @@ void CardboardQrCode_destroy(const uint8_t* encoded_device_params) {
   delete[] encoded_device_params;
 }
 
+void CardboardQrCode_saveDeviceParams(const uint8_t *uri, int size) {
+  if (CARDBOARD_IS_NOT_INITIALIZED() || CARDBOARD_IS_ARG_NULL(uri)) {
+    return;
+  }
+  if (size <= 0) {
+    CARDBOARD_LOGE("Viewer params URI: Size is not valid");
```

Comment:
Consider adding file and line information, e.g.:

CARDBOARD_LOGE("[%s : %d] Argument size is not valid. It must be higher than zero.", __ FILE __, __ LINE __);